examples/widgets/server.py

```diff
@@ -39,6 +40,14 @@ class ChatHistory(BaseModel):
     question: str
 
 
+class ChatHistoryMessage(BaseModel):
+    chat_history: List[BaseMessage] = Field(
+        ...,
+        extra={"widget": {"type": "chat", "input": "question", "output": ""}},
```

Comment:
Can we do this instead?

```suggestion
        extra={"widget": {"type": "chat", "input": "question"}},
```